sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -0,0 +1,70 @@
+# Azure Template Package client library for C++
+
+Azure Template Package client library for C++ (`azure-template`)  matches necessary patterns that the development team has established to create a unified SDK written in the C++ programming language. These libraries follow the Azure SDK Design Guidelines for C++.
```

Comment:
WIP? You might want to remove the template specific content to ensure it doesn't bleed through to the release.

Reply:
Yeah, Readme update is TODO (will file tracking issues).
I will remove the contents for now, leaving the barebones heading.